darts/models/forecasting/tft_model.py

```diff
@@ -288,7 +288,7 @@ def get_relative_index(encoder_length: int,
         prediction_index = encoder_length - 1
         index[:encoder_length] = index[:encoder_length] / prediction_index
         index[encoder_length:] = index[encoder_length:] / prediction_index
-        return index.resize(1, len(index), 1).repeat(batch_size, 1, 1)
```

Comment:
Wait, this was returning None?

Reply:
No, it just had a DeprecationWarning that non-inplace `resize` shouldn't be used anymore